sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuBroadcastNestedLoopJoinExec.scala

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package com.nvidia.spark.rapids.shims.spark311
```

Comment:
Now that `GpuBroadcastNestedLoopJoinExec` is no longer shimmed in any way, should this be moved into the same file as `GpuBroadcastNestedLoopJoinExecBase`?  Is there still a reason for the latter class?

Reply:
there is definitely more things to move around and commonize, I was trying to keep it simple here and easy to review by just moving the files out of shim modules without a lot of code changes that could add bugs and make it harder to review.   We can then follow up with more changes.

Reply:
I'm fine with followups as long as we're tracking somewhere the things we want to do so they don't get dropped.